src/index.ts

```diff
@@ -171,6 +178,11 @@ interface EventLogger {
    * Logs `click`.
    */
   logClick(click: Click): void;
+
+  /**
+   * Flush any Snowplow events that have been queued up in the EventLogger.
+   */
+  tryFlush(): void;
```

Comment:
Just wondering if "tryXyz" is convention in JavaScript/Typescript for an operation that might fail? This is only a question.

Reply:
No convention.  This is just me.  I'll rename it to `flushWrapper`.  It doesn't actually flush Snowplow tracker.  It just flushes the events out of this wrapper.